### PR TITLE
[zh] Translate enforce standards-namespace-labels page

### PR DESCRIPTION
**Pull request by tengqm**
/language zh

### CODE REVIEW

File: content/zh/docs/tasks/configure-pod-container/enforce-standards-namespace-labels.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+- tallclair
+- liggitt
+content_type: task
+min-kubernetes-server-version: v1.22
+-->
+
+<!--
+Namespaces can be labeled to enforce the [Pod Security Standards](/docs/concepts/security/pod-security-standards).
+-->
+你可以通过为名字空间设置标签来强制实施 [Pod 安全标准](/zh/docs/concepts/security/pod-security-standards).
```
Comment (howieyuen) [inline content/zh/docs/tasks/configure-pod-container/enforce-standards-namespace-labels.md:18]: nit, `.` -> `。`

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh/docs/tasks/configure-pod-container/enforce-standards-namespace-labels.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+- _Blocks_ any pods that don't satisfy the `baseline` policy requirements.
+- Generates a user-facing warning and adds an audit annotation to any created pod that does not
+  meet the `restricted` policy requirements.
+- Pins the versions of the `baseline` and `restricted` policies to v{{< skew latestVersion >}}.
+-->
+下面的清单定义了一个 `my-baseline-namespace` 名字空间，其中
+
+- *阻止*任何不满足 `baseline` 策略要求的 Pods；
+- 针对任何无法满足 `restricted` 策略要求的、已创建的 Pod 为用户生产警告信息，
```
Comment (howieyuen) [inline content/zh/docs/tasks/configure-pod-container/enforce-standards-namespace-labels.md:45]:
```suggestion
- 针对任何无法满足 `restricted` 策略要求的、已创建的 Pod 为用户生成警告信息，
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

**Review by tanjunchen — COMMENTED** [review]
/approve

**Pull request merged**